stages/org.osbuild.dnf.config

```diff
@@ -60,13 +78,43 @@ def configure_variable(tree, name, value):
     with open(f"{tree}{vars_directory}/{name}", "w") as f:
         f.write(value + "\n")
 
+def make_value(value):
+    val = str(value)
+    return val
+
+def make_section(cfg, name, settings):
+    if not cfg.has_section(name):
+        cfg.add_section(name)
+
+    for key, value in settings.items():
+        val = make_value(value)
+        cfg.set(name, key, val)
+
+def make_dnf_conf(config):
+    """
+    Merges the given config object into /etc/dnf/dnf.conf, overwriting existing
+    values.
+    """
+    dnfconf = "/etc/dnf/dnf.conf"
```

Comment:
That would read (and write) the config of the build root, what you want is:
```suggestion
    dnfconf = os.path.join(tree, "etc/dnf/dnf.conf")
```
Need to pass in `tree` though.